script.js

```diff
@@ -0,0 +1,13 @@
+const togglePassword = document.querySelector('#toggle-pass')
+const passwordInput = document.querySelector('#pass-input')
+const confirmPasswordInput = document.querySelector('#cpass-input')
+
+togglePassword.addEventListener('change', () => {
+    if(passwordInput.type === 'password') {
+        passwordInput.type = 'text'
+        confirmPasswordInput.type = 'text' 
+    } else if(passwordInput.type === 'text') {
```

Comment:
As there are only two conditions, you can avoid the else if block and use an else block instead.

Reply:
code update.